src/plugins/vis_type_timeseries/common/agg_utis.ts

```diff
@@ -0,0 +1,382 @@
+      ...aggDefaultMeta,
+      isFilterRatioSupported: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.cardinalityLabel', {
+        defaultMessage: 'Cardinality',
+      }),
+    },
+  },
+  {
+    id: METRIC_TYPES.COUNT,
+    meta: {
+      ...aggDefaultMeta,
+      isFieldRequired: false,
+      isFilterRatioSupported: true,
+      isHistogramSupported: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.countLabel', { defaultMessage: 'Count' }),
+    },
+  },
+  {
+    id: TSVB_METRIC_TYPES.FILTER_RATIO,
+    meta: {
+      ...aggDefaultMeta,
+      isFieldRequired: false,
+      label: i18n.translate('visTypeTimeseries.aggUtils.filterRatioLabel', {
+        defaultMessage: 'Filter Ratio',
+      }),
+    },
+  },
+  {
+    id: TSVB_METRIC_TYPES.POSITIVE_RATE,
+    meta: {
+      ...aggDefaultMeta,
+      isFilterRatioSupported: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.positiveRateLabel', {
+        defaultMessage: 'Counter Rate',
+      }),
+    },
+  },
+  {
+    id: METRIC_TYPES.MAX,
+    meta: {
+      ...aggDefaultMeta,
+      isFilterRatioSupported: true,
+      isHistogramSupported: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.maxLabel', { defaultMessage: 'Max' }),
+    },
+  },
+  {
+    id: METRIC_TYPES.MIN,
+    meta: {
+      ...aggDefaultMeta,
+      isFilterRatioSupported: true,
+      isHistogramSupported: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.minLabel', { defaultMessage: 'Min' }),
+    },
+  },
+  {
+    id: TSVB_METRIC_TYPES.PERCENTILE,
+    meta: {
+      ...aggDefaultMeta,
+      label: i18n.translate('visTypeTimeseries.aggUtils.percentileLabel', {
+        defaultMessage: 'Percentile',
+      }),
+    },
+  },
+  {
+    id: TSVB_METRIC_TYPES.PERCENTILE_RANK,
+    meta: {
+      ...aggDefaultMeta,
+      label: i18n.translate('visTypeTimeseries.aggUtils.percentileRankLabel', {
+        defaultMessage: 'Percentile Rank',
+      }),
+    },
+  },
+  {
+    id: TSVB_METRIC_TYPES.STATIC,
+    meta: {
+      ...aggDefaultMeta,
+      isFieldRequired: false,
+      label: i18n.translate('visTypeTimeseries.aggUtils.staticValueLabel', {
+        defaultMessage: 'Static Value',
+      }),
+    },
+  },
+  {
+    id: TSVB_METRIC_TYPES.STD_DEVIATION,
+    meta: {
+      ...aggDefaultMeta,
+      hasExtendedStats: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.deviationLabel', {
+        defaultMessage: 'Std. Deviation',
+      }),
+    },
+  },
+  {
+    id: METRIC_TYPES.SUM,
+    meta: {
+      ...aggDefaultMeta,
+      isFilterRatioSupported: true,
+      isHistogramSupported: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.sumLabel', { defaultMessage: 'Sum' }),
+    },
+  },
+  {
+    id: TSVB_METRIC_TYPES.SUM_OF_SQUARES,
+    meta: {
+      ...aggDefaultMeta,
+      hasExtendedStats: true,
+      label: i18n.translate('visTypeTimeseries.aggUtils.sumOfSquaresLabel', {
+        defaultMessage: 'Sum of Squares',
```

Comment:
Shouldn't this be `Sum of squares`?

Reply:
this label is picked from [_agg_select.tsx_](https://github.com/elastic/kibana/pull/109424/files#diff-a914412c0831d5fc1e0ceb6b8d7bb328375a20b19292a9a3438ba7f5382f4f86L94), the same is for `Overall Sum of Squares`
![image](https://user-images.githubusercontent.com/54894989/130605003-c7b19205-f633-4ea9-acbf-88ddef15bc89.png)